src/plus/webviews/graph/graphWebview.ts

```diff
@@ -1410,9 +1411,12 @@ export class GraphWebview extends WebviewBase<State> {
 		};
 	}
 
-	private updateColumn(name: GraphColumnName, cfg: GraphColumnConfig) {
+	private updateColumns(columnsCfg: GraphColumnsConfig) {
 		let columns = this.container.storage.getWorkspace('graph:columns');
-		columns = updateRecordValue(columns, name, cfg);
+		Object.keys(columnsCfg).forEach((key: string) => {
+			const name = key as GraphColumnName;
+			columns = updateRecordValue(columns, name, columnsCfg[name]);
+		});
```

Comment:
In general, I avoid `forEach` loops unless we aren't creating a new closure. As they are harder for the runtime to optimize (mainly because a new closure is created) and harder to step into vs a `for...of` loop.

I'd also rewrite this to avoid an additional lookup on each loop iteration.
```suggestion
		for (const [key, value] of Object.entries(columnsCfg)) {
			columns = updateRecordValue(columns, key, value);
		});
```

Reply:
Done in commit: `Avoid "forEach" loops `